app/Policies/RankPolicy.php

```
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Policies;
+
+use App\Models\Enums\UnitPermission;
+use App\Models\Rank;
+use App\Models\User;
+
+class RankPolicy
+{
+    public function viewAny(?User $user): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$user'. (undefined)

(UnusedFormalParameter)